optimization/matRad_IpoptIterFunc.m

```diff
@@ -65,7 +65,10 @@
     hold on, grid on, grid minor,
     AxesInfigOpt = findall(figOpt,'type','axes');
 end
-
+% ensure to bring optimization window to front also for a re-optimization 
+if isdeployed
```

Comment:
focus is only set to the iterations window if matRad is deployed. matRad forces the window on each iteration to be shown on top of the GUI when running the standalone version. Matlab sets the focus to the window but it does not change the global focus which means that other programs can used meanwhile.